src/platform/mt793x/DnssdImpl.cpp

```diff
@@ -273,5 +273,10 @@ CHIP_ERROR ChipDnssdResolve(DnssdService * service, chip::Inet::InterfaceId inte
     return error;
 }
 
+CHIP_ERROR ChipDnssdReconfirmRecord(const char * hostname, chip::Inet::IPAddress address, chip::Inet::InterfaceId interface)
```

Comment:
Having most platforms being CHIP_ERROR_NOT_IMPLEMENTED seems painful. How about ifdefing in this case and only calling it for apple/darwin?